frb_codegen/src/others.rs

```diff
@@ -47,7 +55,8 @@ pub fn modify_dart_wire_content(content_raw: &str, dart_wire_class_name: &str) -
     let content = content_raw.replace(
         &format!("class {} {{", dart_wire_class_name),
         &format!(
-            "class {} implements FlutterRustBridgeWireBase {{",
+            "class {} implements FlutterRustBridgeWireBase {{
+            late final dartApi = DartApiDl(init_frb_dart_api_dl);",
```

Comment:
uber-nit: Btw shall it be @internal or private?

Reply:
Yes, this is an `internal` field, but end users don't have access to it anyway.
(added attribute `internal`)